test/lib/rules/disallow-injectedcontaineraccess.js

```diff
@@ -0,0 +1,58 @@
+describe('lib/rules/disallow-injectedcontaineraccess', function () {
+    var checker = global.checker({
+        plugins: ['./lib/index']
+    });
+
+    describe('not configured', function() {
+
+      it('should report with undefined', function() {
+        global.expect(function() {
+          checker.configure({disallowInjectedContainerAccess: undefined});
+        }).to.throws(/requires a true value/i);
```

Comment:
It'd be great to change these messages to `requires a boolean value` to match conventions. See https://github.com/minichate/jscs-ember-deprecations/blob/master/test/lib/rules/disallow-defaultlayout.js#L11 as an example.